app/batid/models/building.py

```diff
@@ -550,3 +551,11 @@ class Meta:
             models.Index(fields=("event_type",), name="bdg_history_event_type_idx"),
             GinIndex(fields=["parent_buildings"], name="bdg_hist_parent_buildings_idx"),
         ]
+
+        constraints = [
+            # verify the couple rnb_id/event_id is unique
+            UniqueConstraint(
+                fields=["rnb_id", "event_id"],
+                name="unique_rnb_id_event_type",
```

Comment:
Le nom ne correspond pas à la contrainte
```suggestion
                name="unique_rnb_id_event_id",
```